bin/prettier.js

```diff
@@ -171,17 +171,15 @@ if (stdin) {
 } else {
   eachFilename(filepatterns, filename => {
     fs.readFile(filename, "utf8", (err, input) => {
-      if (write) {
-        console.log(filename);
-      }
-
       if (err) {
```

Comment:
What's more likely is that the formatting will fail for some reason because of a bug. I really want to simply print the filename like we did before.

What we can do instead is print the filename with `process.write` or whatever the call is to write straight to the buffer without adding a newline. Then you can add additional info after the file has been processed on the same line, and add the newline yourself.